pkgs/applications/editors/rstudio/default.nix

```diff
@@ -62,8 +63,8 @@ let
   rsconnectSrc = fetchFromGitHub {
     owner = "rstudio";
     repo = "rsconnect";
-    rev = "5175a927a41acfd9a21d9fdecb705ea3292109f2";
-    hash = "sha256-c1fFcN6KAfxXv8bv4WnIqQKg1wcNP2AywhEmIbyzaBA=";
+    rev = "872b53e3d39a43c543498f437587fd7fb7fdca59";
```

Comment:
I meant to leave a comment here instead, sorry :sweat_smile: 

https://github.com/NixOS/nixpkgs/pull/289393#pullrequestreview-1887058463